### ISSUE TITLE
Payments Research: Adoption of Benefits Program, Senior Discount

### ISSUE THREAD

**Issue by vykster** [cal-itp/data-infra#951]
## Question

Is the Cal-ITP Benefits application successfully reaching and enrolling riders who are then riding and paying a discounted-fare?

## Metrics

_Very open to other suggestions on metrics. Our goal is to programmatically capture at least one measure that indicates use of the discount program._

Suggested:
* % of total fares processed as a discount per time period (day, month, etc)

## Data sources

* Littlepay Merchant Data Feed API [API docs](https://drive.google.com/file/d/1QcGL-tVoan30RL9Hbv-scLtrI5dG9IYr/view)
* Values needed: product_id or description
 * Littlepay product_ids and descriptions for MST: 
 https://github.com/cal-itp/benefits-secrets/tree/main/littlepay/prod/mst

## Additional Notes from Requester

* The Benefits application places a payment card into a "group" and "products" are linked to those groups. At first glance it seems that groups are not available in the data feed, but a "product_id" is available. _Multiple discount products are configured for each agency? Yes, MST has a four discount fare product that has as single ride, daily, weekly, and monthly discount cap. _
* This metric could potentially calculated by investigating the $ spent, but that would be less accurate than using the product_id.
* Currently all riders enrolled are seniors, however this will change shortly and it's expected that agencies and Cal-ITP staff would like to see the breakdown by program (ie: Veterans vs Senior)
* ~~When poking around metabase: Across multiple discount-fare products -- it seems the Product description "Discount-fare" might be useful? At this point I am unable to tell if this is the description for a single product or used for multiple products. ~~ It seems this is the name (product description)of the MST fare-discount product. 
* Note, The benefits dev team is manually pulling web analytics for the app and posting them to slack on Fridays. Amplitude is being used to capture site analytics. 
* This metric may already be available to us in metabase by filtering on the [Number of Reduced-Price Rides Per Week, Grouped by Product Code](https://dashboards.calitp.org/question/183-number-of-reduced-price-rides-per-week-grouped-by-product-code). This visualization captures all adjustments, not just those associated with the discount-fare product
* Q to ask metrics team: How are these best accessible to the benefits team and the client MST? 
* Q: is it appropriate to have/ create a table to show product ids, names/description, agency they are associated with, etc?

## (Data Services Team to Copy and Fill Out Below)

The QuVR MD template below will be filled out by a member of the data services team.
This allows us to describe the request, in a way that is easy to hand-off for analysis.
After the research phase, we will sync with the asker to figure out if the metric and dashboard pieces are needed.

Before starting research:

* **Question**: Question written as a single sentence.
* **View**: E.g. `views.gtfs_schedule_fact_daily_feed_files`.
* **Research**:
  - How should the results be presented?
  - When are they needed by?

After reviewing research with the asker:

* **Metric**: what specific calculations are needed?
* **Dashboard**: where should we put the result?

Todo

- [ ] submit ticket to Littlepay to request groups are added to Data Feed API

**Comment by vykster** [comment]
@mjumbewu would you have a few minutes to walk me through some metabase and payments data questions? --> https://calendly.com/vykster

**Comment by mjumbewu** [comment]
Sure @vykster. How urgent? Your Tuesday slots seem most reasonable for me, but if it's more urgent I could probably swing Friday.

**Comment by vykster** [comment]
Not urgent, Tuesday sounds good

**Comment by vykster** [comment]
Note to Vyki, need to submit ticket to Littlepay to request groups are added to Data Feed API

**Comment by vykster** [comment]
@holly-g could you assign this issue to me and/or give me permission to assign issues?

**Comment by holly-g** [comment]
@vykster I don't believe I have the privileges to grant you permission (I will ask @evansiroky to do so), but I just assigned this issue to you

**Comment by indexing** [comment]
@vykster I believe we can close this issue?

**Comment by vykster** [comment]
sounds good it was not solved but as it's not an urgent need or priority we
can make a new issue in time. - vyki
